templates/app/server/api/user(auth)/user.model(mongooseModels).js

```diff
@@ -95,10 +95,6 @@ UserSchema
   .path('email')
   .validate(function(value, respond) {
     var self = this;
-    <%_ if(filters.oauth) { _%>
-    if (authTypes.indexOf(this.provider) !== -1) {
-      return respond(true);
-    }<% } %>
```

Comment:
Sorry in my case it is redundant because i want to check email in case of social login that it is already available or not in the DB.